docs/ancestry.rst

```diff
@@ -748,10 +731,92 @@ rate along the genome can be combined in ``msprime``.
     import msprime
     from IPython.display import SVG
 
-.. todo:: Examples of small simulations using GC on its own
-    and mixed with recombination. Ideally we'd start with an
-    example with one GC event where we could explain what happened.
-    This might be tricky to finesse.
+Gene conversion events are defined by two parameters. 
+For discrete genomes these are:
+The rate at which gene conversion events are initiated between two
+adjacent sites and the mean of the geometrically distributed 
+gene conversion tract lengths. Note that if the mean gene conversion
+tract length is set to 1 all tract lengths will be of length 1.
+In the following example one gene conversion event has occured.
+
+
+.. jupyter-execute::
+
+    ts = msprime.sim_ancestry(
+        3, gene_conversion_rate=0.02, gene_conversion_tract_length=1,
+        sequence_length=10, random_seed=3)
+    ts.sequence_length
+    SVG(ts.draw_svg())
+
+.. jupyter-execute::
+    :hide-code:
+
+    assert 1 < ts.num_trees < 5
+    
+Continous genomes can also be used. In this case the parameters define
+the rate at which gene conversion events are initiated per unit of 
+sequence length and the mean of the exponentially distributed gene 
+conversion tract lengths. It is important to use a sequence_length
+that corresponds to the actual number of sites considered or to scale
+the gene conversion rate and tract length accordingly. Otherwise the
+distribution of the tract_lengths will not be correct. Note that if the 
+mean gene conversion tract length is set to 1 all tract lengths will be
+almost sure different. However, all gene conversion events should be
```

Comment:
I think these four sentences will just confuse people to be honest - these are the same considerations as you would have  when simulating recombination or mutation on continuous genome, and we don't really need to get into it in particular for GC. I think the example will make things clear enough, and the how we should interpret continuous coordinates dealt with in a separate section which considers GC, recombination and mutation at the same time.

So, I would delete everything from "...tract lenghts." to "The following example..."

Reply:
Done.